### DISCUSSION TITLE
Using without Unity?

### DISCUSSION THREAD

**Discussion by 0xF6** [ialex32x/unity-jsb#88] [Q&A]
I'm looking for an engine for mods that will be well integrated into unity, but at the same time so that it can be installed in a regular console application (server .NET5-6\netStandart and etc)

After skimming through the code, I didn't find the answer is it possible to run jsb without unity?
Of course, I don't need api unity on the server

**Comment by 0xF6** [comment]
Ah, i see JSB_UNITYLESS, but I would still like to hear comments on this

**Reply by ialex32x** [reply]
Yes, JSB_UNITYLESS is the one for using without unity, such as using in dotnetcore. There is a simple example in `jsb_build/dotnetcore`.  But it's not fully tested (the whole project unity-jsb is not fully tested by the way 😂).

**Reply by 0xF6** [reply]
Oke thanks)
Are you planning to release a nuget unityless package?

**Reply by ialex32x** [reply]
Not yet, but will try if it's not too difficult.
